test/unit/entry.spec.js

```diff
@@ -50,7 +51,7 @@ describe(`createEntry()`, () => {
       })
       it(`got instantiated with its related constructor given field info`, () => {
         Object.getOwnPropertyNames(entry.fields).forEach((fieldId) => {
-          const info = fieldInfo.find((info) => info.id === fieldId)
+          const info = find(fieldInfo, (info) => info.id === fieldId)
```

Comment:
Why not stick with `Array.prototype.find`? A whole new dependency just to use `_.find` seems too much imho.